src/scripts.js

```diff
@@ -160,6 +176,14 @@ window.addEventListener('load', displayCurrentRecipes);
 
 filterButton.addEventListener('click', clickFilterButton);
 
+homeButton.addEventListener('click', clickHomeButton);
```

Comment:
I love the difference in event listener functionality between the home button, filter button being unclicked and the logo refreshing the page.